system/system.proto

```diff
@@ -311,3 +314,20 @@ message SetPackageRequest {
 
 message SetPackageResponse {
 }
+
+// RestartProcessRequest describes the process restart operation.
+message RestartProcessRequest {
+  // Name of the process to be restarted.
+  string name = 1;
+  // Mode of restarting.
+  enum Mode {
```

Comment:
Should these map to os signals?
https://man7.org/linux/man-pages/man7/signal.7.html
or at least use the same names or ?

Reply:
lgtm